src/App.tsx

```diff
@@ -1,7 +1,38 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string;
+};
+
+export class App extends React.Component {
+  state: State = {
+    pressedKey: '',
+  };
+
+  // eslint-disable-next-line react/no-deprecated
+  componentWillMount(): void {
```

Comment:
componentWillMount is deprecated. Use componentDidMount
```suggestion
  componentDidMount(): void {
```